test/controllers/GenericExplorerControllerSpec.scala

```diff
@@ -110,5 +115,113 @@ class GenericExplorerControllerSpec extends PlaySpec with OneAppPerSuite with Mo
 
     }
   }
+
+  "The property search controller" should {
+    "return the nodes of a given property if they exist" in {
+      importStorage
+
+      val prop = "importance"
+      val t = g.V().has( prop ).asScala.toList
+
+      val result = genericController.retrieveNodeProperty( prop ).apply( fakerequest )
+      val content = contentAsJson( result ).as[List[PersistedVertex]]
+
+      content.length == t.length mustBe true
+
+    }
+  }
+
+  "The property search controller" should {
+    "be able to return a list of 1" in {
+      importStorage
+      val prop = "test"
+      val t = g.V().has( prop ).asScala.toList
+
+      val result = genericController.retrieveNodeProperty( prop ).apply( fakerequest )
+      val content = contentAsJson( result ).as[List[PersistedVertex]]
+
+      content.length == t.length mustBe true
+
+    }
+  }
+
+  "The property search controller" should {
+    "return an empty list" in {
+
+      val prop = "month"
+
+      val result = genericController.retrieveNodeProperty( prop ).apply( fakerequest )
+      val content = contentAsJson( result ).as[List[PersistedVertex]]
+
+      content.length mustBe 0
+    }
+  }
+
+  "The value search controller" should {
+    "return all values for a property in a list" in {
+      importStorage
+
+      val prop = "importance"
```

Comment:
Replace `importance` by `system:creation_time` and it fails

Reply:
That is surprising as I explicitely put "values[String]". I will investigate.

Reply:
Fixed.